lib/hon_graffiti_phoenix/parsers/markup_parser.ex

```diff
@@ -28,9 +28,9 @@ defmodule HonGraffitiPhoenix.Parsers.MarkupParser do
   If no markup is found, a default is given.
   """
   @spec parse(String.t) :: [DecoratedString]
-  def parse(quote) when quote == nil, do: []
-  def parse(quote) do
-    quote
+  def parse(hon_quote) when hon_quote == nil, do: []
+  def parse(hon_quote) do
+    hon_quote
```

Comment:
We'll also want to change all other references, including the model name... Don't need to do right now, I suppose.